src/prefect/agent/kubernetes/README.md

```diff
@@ -0,0 +1,11 @@
+# k8s-agent
+
+The Prefect Kubernetes agent that turns a cluster into a workflow orchestration system.
+
+If running on GKE you may need to execute: `kubectl create clusterrolebinding default-admin --clusterrole cluster-admin --serviceaccount=default:default`
+
+Quick Start:
+
+- Build Dockerfile and push to registry
+- Update the `image` and `PREFECT__CLOUD__AUTH_TOKEN` env value in the deployment yaml
+- Run `kubectl apply -f deployment.yaml`
```

Comment:
Should we introduce a `prefect agent deploy k8s --image= --token=` CLI command to take care of this (in a future PR)?

Reply:
Yeah I have a whole plan for added k8s functionality from the CLI, this being one of them